cdk/lib/auth/auth-stack.ts

```diff
@@ -18,20 +19,46 @@ export class AuthStack extends Stack {
   readonly identityPool: CfnIdentityPool;
   readonly userPool: UserPool;
   readonly userPoolClient: UserPoolClient;
+  readonly domain: UserPoolDomain;
 
   constructor(scope: Construct, id: string, props: AuthStackProps) {
     super(scope, id, props);
 
     const { applicationName, logGroupArn, removalPolicyOverride } = props;
 
+    const authMode = this.node.tryGetContext('authMode') as string;
```

Comment:
`tryGetContext` returns type `any` so if I don't cast this will fail the linter rule for `Unsafe assignment of an any value`. You're right about the existence check it's not necessary, authMode should always exist or else the deployment should fail, so I removed the existence check.